src/javascripts/app.js

```diff
@@ -1,3 +1,5 @@
-import './modules'
+import "./modules";
+
+import "../../node_modules/svgxuse/svgxuse";
```

Comment:
should be able to just use `import "svgxuse"`

the magic of webpack is that it will look at the entry point file specified by the svgxuse package. so it tracks down the file for you.